### DISCUSSION TITLE
Is there a way to get Koka syntax highlighting in any editor?

### DISCUSSION THREAD

**Discussion by BrendanMesters** [koka-lang/koka#262] [Q&A]
I'm going to write a relatively large project in Koka and was wondering if there is any way to do syntax highlighting in an editor (I'd probably even switch editor for that). I haven't really started programming yet, but already noticed that I couldn't find a way to have syntax highlighting, nor could I find any solutions online.

Help would be much appreciated :-D

**Comment by timjs** [comment]
You can find editor support in the `support` folder of the repository. The VSCode one is also available from within the app!
